File: erlang/fizzbuzz.erl
-module(fizzbuzz).

calculate([]) -> "";
calculate([X|Xs]) when (X rem 3 == 0) and (X rem 5 == 0) -> "FizzBuzz\n" ++ calculate(Xs);
calculate([X|Xs]) when X rem 3 == 0 -> "Fizz\n" ++ calculate(Xs);
calculate([X|Xs]) when X rem 5 == 0 -> "Buzz\n" ++ calculate(Xs);
calculate([X|Xs]) -> integer_to_list(X) ++ "\n" ++ calculate(Xs).

main(_) -> io:format(calculate(lists:seq(1, 100))).

File: perl/fizzbuzz.pl
foreach (1..100) {

	if ($_ % 3 == 0) {
		print "Fizz";
	}

	if ($_ % 5 == 0) {
		print "Buzz";
	}

	if (($_ % 3 != 0) && ($_ % 5 != 0)) {
		print $_;
	}


	print "\n";
}
